grammar.y

```diff
@@ -251,17 +250,47 @@ semicolon_opt:
 ;
 
 select_stmt:
```

Comment:
the only thing to pay attention to here is that the `base_select` cannot have `order by` and `limit`. this is analogous to [select core](https://www.sqlite.org/syntax/select-core.html)